examples/core/list_exports/src/main.rs

```diff
@@ -0,0 +1,87 @@
+use frida::{Frida, Message};
+use lazy_static::lazy_static;
+use std::{thread, time::Duration};
+
+lazy_static! {
+    static ref FRIDA: Frida = unsafe { Frida::obtain() };
+}
+
+fn main() {
+    let pid = 1134009;
+    let device_manager = frida::DeviceManager::obtain(&FRIDA);
+    let local_device = device_manager.get_device_by_type(frida::DeviceType::Local);
+
+    if let Ok(device) = local_device {
+        println!("[*] Frida version: {}", frida::Frida::version());
+        println!("[*] Device name: {}", device.get_name());
+
+        // Attach to the program
+        let session = device.attach(pid).unwrap();
+
+        if session.is_detached() {
+            println!("Session is detached");
+            return;
+        }
+
+        println!("[*] Attached to PID {}", pid);
+
+        let script_source = r#"
+            var globalVar = 0;
+            console.log("Logging message from JS");
+            console.warn("Warning message from JS");
+            console.debug("Debug message from JS");
+            console.error("Error message from JS");
+
+            rpc.exports = {
+                increment: function() {
+                    globalVar += 1;
+                    return globalVar;
+                },
+                getvalue: function() {
+                    return globalVar;
+                }
+            };
+
+            ;sdfsa // <- Intentional error here
```

Comment:
What is the error for?

Reply:
I wanted to show the different types of messages. That's why in the readme I added the output to expect. I think it is easier to show this kind of things for newcomers, but I can remove it if you want to.

Reply:
I don't like it. Let's get rid of it. If there is another way to show this, do so.

Reply:
There is not other way to trigger a JS error without having one. I removed it.